### PR TITLE
Update main to use Fluent Scheduler and sea-level use cases

### PR DESCRIPTION
**Pull request by chubberlisk**

### CODE REVIEW

File: CryptoTechReminderSystem.Main/Program.cs [hunk 1]
```diff
@@ -1,43 +1,97 @@
...
         {    
             DotEnv.Config(false);
-            
-            var remindDeveloper = new SendReminder(
-                new SlackGateway(
-                    "https://slack.com/",
-                    Environment.GetEnvironmentVariable("SLACK_TOKEN")
-                )
+            JobManager.Initialize(new ReminderRegistry());
+            SpinWait.SpinUntil(() => false);
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Main/Program.cs]: Why do we need this?

File: CryptoTechReminderSystem.Main/Program.cs [hunk 2]
```diff
@@ -1,43 +1,97 @@
...
+        public ReminderRegistry()
+        {
+            var slackGateway = new SlackGateway(
+                "https://slack.com/",
+                Environment.GetEnvironmentVariable("SLACK_TOKEN")
+            );
+            var harvestGateway = new HarvestGateway(
+                "https://api.harvestapp.com/",
+                Environment.GetEnvironmentVariable("HARVEST_TOKEN"),
+                Environment.GetEnvironmentVariable("HARVEST_ACCOUNT_ID")
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Main/Program.cs]: The request header for the `UserAgent` can also be an environment variable which also needs to be passed in as an argument here.

File: CryptoTechReminderSystem.Main/Program.cs [hunk 3]
```diff
@@ -1,43 +1,97 @@
...
+        private void CreateSchedule()
+        {
+            // Creates schedule that starts and stops schedule jobs
+            JobManager.AddJob(ScheduleJobs, s => s.ToRunEvery(1).Weeks().On(DayOfWeek.Friday).At(10,30));
+            JobManager.AddJob(ResetSchedule, s => s.ToRunEvery(1).Weeks().On(DayOfWeek.Friday).At(13,45));
+        }
+
+        private void ScheduleJobs()
+        {
+            // Do all of your actual job scheduling here
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Main/Program.cs]: Unnecessary commenting

File: CryptoTechReminderSystem.Main/Program.cs [hunk 4]
```diff
@@ -1,43 +1,97 @@
...
+        }
+        
+        private void ShameLateDevelopersJob()
+        {
+            var shameLateDevelopers = new ShameLateDevelopers(_getLateDevelopers, _sendReminder, _clock);
+            shameLateDevelopers.Execute(
+                new ShameLateDevelopersRequest
+                {
+                    Message = "These are the people yet to submit timesheets:",
+                    Channel = "CGHBW6YGM"
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Main/Program.cs]: How do we know what channel this is? Could this be an environment variable as well?

File: CryptoTechReminderSystem.Main/Program.cs (hunk 1 above)
Comment (RobertM905) [reply CryptoTechReminderSystem.Main/Program.cs]: It stops the main from exiting when all the schedules are setup. We want the program to keep running.

File: CryptoTechReminderSystem.Main/Program.cs (hunk 4 above)
Comment (RobertM905) [reply CryptoTechReminderSystem.Main/Program.cs]: 👍

File: CryptoTechReminderSystem.Main/Program.cs (hunk 1 above)
Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Main/Program.cs]: Agreed with Robert!

File: CryptoTechReminderSystem.Main/Program.cs (hunk 2 above)
Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Main/Program.cs]: Added environment variable for user agent in https://github.com/chubberlisk/crypto_tech/pull/25/commits/4aa9b651c7633661a3869eed2ac13696502033a1.

File: CryptoTechReminderSystem.Main/Program.cs (hunk 3 above)
Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Main/Program.cs]: Resolved in https://github.com/chubberlisk/crypto_tech/pull/25/commits/8432cfb2334ce8c2878f5084f4db4d2526bcc8c7.

File: CryptoTechReminderSystem.Main/Program.cs (hunk 4 above)
Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Main/Program.cs]: Resolved in https://github.com/chubberlisk/crypto_tech/pull/25/commits/9345d3c8934d428d33b5c6be1f01accb018def22.

**Pull request merged**